src/main/java/uk/nhs/prm/e2etests/property/DynamoDbProperties.java

```diff
@@ -4,19 +4,24 @@
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Component;
 
+/**
+ * @deprecated PRMT-4670 old Transfer Tracker DB logic - to be removed
+ */
+@Deprecated(since="2.0.0")
 @Getter
 @Component
 public class DynamoDbProperties {
-    /**
-     * @deprecated PRMT-4670 old Transfer Tracker DB logic - to be removed
-     */
-    @Deprecated(since="2.0.0", forRemoval = true)
+    @Deprecated(since="2.0.0")
     @Value("${aws.configuration.databaseNames.oldTransferTrackerDb}")
     private String oldTransferTrackerDbName;
 
+    // TODO - PRMT-4663 - USE @Value WHERE IT IS REQUIRED,
+    // TODO - PRMT-4663 - LET'S NOT COUPLE THESE NAMES!
     @Value("${aws.configuration.databaseNames.transferTrackerDb}")
     private String transferTrackerDbName;
 
+    // TODO - PRMT-4663 - USE @Value WHERE IT IS REQUIRED,
+    // TODO - PRMT-4663 - LET'S NOT COUPLE THESE NAMES!
```

Comment:
Should these be addressed with this ticket, PRMT-4663 is the parent!

Reply:
I've removed these TODO messages, they're on a deprecated Component that is only used in logic that is to be removed, and the active suspensions (which is also going to be removed eventually).